tests/es6_example/follow-link.js

```diff
@@ -0,0 +1,12 @@
+const expect = require('chai').expect;
+const HomePage = require('../../pages/es6_example/home.page');
+const homePage = new HomePage();
+
+describe('Mocha Spec Sync example', function() {
+
+    it("follows link to new page", function() {
+        homePage.open();
+        homePage.follow_link();
```

Comment:
Sorry man, I'm a clean code stickler and not sure if it applies here. But should methods in JS be Camel Case? I see one method called setComment() and another called follow_link(). Sorry if it's a stupid question, relatively new to JS.

Reply:
Following in the fine tradition of PHP, it's appropriate to mix camelCase and underscore (but I believe the preferred term is lowdash, these days).

Reply:
Great question! I'm not sure what the conventions are for NodeJS code. I do know that there are some common style guides/linters like the [AirBnB style guide](https://github.com/airbnb/javascript). I feel like we didn't apply any style guides here due to scope of the project.

Reply:
How about this - https://docs.npmjs.com/misc/coding-style
Also, I think whatever convention we choose, it should be consistent if we're doing best practices for customers